pkgs/applications/video/mpv/scripts/smart-copy-paste-2.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildLua,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+buildLua (finalAttrs: {
+  pname = "smart-copy-paste-2";
+  version = "unstable-2024-05-13";
+
+  scriptPath = "scripts/SmartCopyPaste_II.lua";
+  src = fetchFromGitHub {
+    owner = "Eisa01";
+    repo = "mpv-scripts";
+    rev = "48d68283cea47ff8e904decc9003b3abc3e2123e";
+    hash = "sha256-95CAKjBRELX2f7oWSHFWJnI0mikAoxhfUphe9k51Qf4=";
+  };
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
Running this update script errors out for me:
```
error:
       … while evaluating attribute 'filename'

         at «string»:23:3:

           22|   inherit raw_version_position;
           23|   filename = position.file;
             |   ^
           24|   line = position.line;

       … while selecting an attribute

         at «string»:23:14:

           22|   inherit raw_version_position;
           23|   filename = position.file;
             |              ^
           24|   line = position.line;

       error: value is null while a set was expected
Traceback (most recent call last):
  File "/nix/store/63zkb5jlwkmgw0xmida3r6pwahbjyx88-nix-update-1.3.1/bin/.nix-update-wrapped", line 9, in <module>
    sys.exit(main())
             ^^^^^^
  File "/nix/store/63zkb5jlwkmgw0xmida3r6pwahbjyx88-nix-update-1.3.1/lib/python3.11/site-packages/nix_update/__init__.py", line 291, in main
    package = update(options)
              ^^^^^^^^^^^^^^^
  File "/nix/store/63zkb5jlwkmgw0xmida3r6pwahbjyx88-nix-update-1.3.1/lib/python3.11/site-packages/nix_update/update.py", line 339, in update
    package = eval_attr(opts)
              ^^^^^^^^^^^^^^^
  File "/nix/store/63zkb5jlwkmgw0xmida3r6pwahbjyx88-nix-update-1.3.1/lib/python3.11/site-packages/nix_update/eval.py", line 205, in eval_attr
    res = run(cmd)
          ^^^^^^^^
  File "/nix/store/63zkb5jlwkmgw0xmida3r6pwahbjyx88-nix-update-1.3.1/lib/python3.11/site-packages/nix_update/utils.py", line 37, in run
    return subprocess.run(
           ^^^^^^^^^^^^^^^
  File "/nix/store/k3701zl6gmx3la7y4dnflcvf3xfy88kh-python3-3.11.9/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['nix', 'eval', '--json', '--impure', '--expr', '\nlet\n  \n  pkgs = import "/home/puna/Development/nixpkgs";\n  args =  builtins.functionArgs pkgs;\n  inputs = (if args ? system then { system = builtins.currentSystem; } else {}) //\n           (if args ? overlays then { overlays = [ ]; } else {});\n  pkg = (pkgs inputs)."mpvScripts"."smart-copy-paste-2";\n  sanitizePosition = x: x;\n\n  raw_version_position = sanitizePosition (builtins.unsafeGetAttrPos "version" pkg);\n\n  position = if pkg ? isRubyGem then\n    raw_version_position\n  else if pkg ? isPhpExtension then\n    raw_version_position\n   else\n    sanitizePosition (builtins.unsafeGetAttrPos "src" pkg);\nin {\n  name = pkg.name;\n  old_version = pkg.version or (builtins.parseDrvName pkg.name).version;\n  inherit raw_version_position;\n  filename = position.file;\n  line = position.line;\n  urls = pkg.src.urls or null;\n  url = pkg.src.url or null;\n  rev = pkg.src.rev or null;\n  hash = pkg.src.outputHash or null;\n  go_modules = pkg.goModules.outputHash or null;\n  go_modules_old = pkg.go-modules.outputHash or null;\n  cargo_deps = pkg.cargoDeps.outputHash or null;\n  raw_cargo_lock =\n    if pkg ? cargoDeps.lockFile then\n      let\n        inherit (pkg.cargoDeps) lockFile;\n        res = builtins.tryEval (sanitizePosition {\n          file = toString lockFile;\n        });\n      in\n      if res.success then res.value.file else false\n    else\n      null;\n  composer_deps = pkg.composerRepository.outputHash or null;\n  npm_deps = pkg.npmDeps.outputHash or null;\n  yarn_deps = pkg.offlineCache.outputHash or null;\n  maven_deps = pkg.fetchedMavenDeps.outputHash or null;\n  tests = builtins.attrNames (pkg.passthru.tests or {});\n  has_update_script = pkg.passthru.updateScript or null != null;\n  src_homepage = pkg.src.meta.homepage or null;\n  changelog = pkg.meta.changelog or null;\n  maintainers = pkg.meta.maintainers or null;\n}', '--extra-experimental-features', 'flakes nix-command']' returned non-zero exit status 1.
```

Reply:
I didn't mean to add it, I forgot to remove it while pushing the new changes...